src/box/iproto.cc

```diff
@@ -3431,3 +3589,56 @@ iproto_thread_rmean_foreach(int thread_id, void *cb, void *cb_ctx)
 		rc = 1;
 	return rc;
 }
+
+int
+iproto_override(uint32_t rq_type, iproto_rq_handler_f cb,
+		iproto_rq_handler_dtor_f dtor, void *ctx)
```

Comment:
Usually, we name destructor functions `destroy` or `free`.

Reply:
Fixed.